src/database/mongodb.ts

```diff
@@ -0,0 +1,15 @@
+const mongoose = require('mongoose');
+
+const ENDPOINT = process.env.ENDPOINT_DB;
+const DESC = process.env.DESC_DB;
```

Comment:
We don't have an env file right now so this will have to contain an OR operator to default

`const ENDPOINT = process.env.ENDPOINT_DB || 'localhost:27017'` should do it.

I also used DESC to log the endpoint but its not too important so I would probably just delete it.